src/networks.json

```diff
@@ -122,6 +122,53 @@
     "start": 657806,
     "logo": "ipfs://QmfF4kwhGL8QosUXvgq2KWCmavhKBvwD6kbhs7L4p5ZAWb"
   },
+  "14": {
+    "key": "14",
+    "name": "Flare Mainnet",
+    "shortName": "Flare",
+    "chainId": 14,
+    "network": "flare",
+    "multicall": "0x0c7B6FC9ffaBC273E9934383f258294A00Db7Fc3",
```

Comment:
Hello @RareData Verify and publish contract code [here ](https://flare-explorer.flare.network/address/0x0c7B6FC9ffaBC273E9934383f258294A00Db7Fc3/contracts) plz :)